### PR TITLE
DOC-4569 - PrimusX version updates. 4.10 Enterprise users can use fir…

### PR DESCRIPTION
**Pull request by paulmoloneyr3**
PrimusX version updates. 4.10 Enterprise users can use firmware versions 2.7.4 or newer for all keys, except for Confidential identity keys which require firmware versions 2.8.5 or newer.

### CODE REVIEW

**Review by colin-hillis — DISMISSED** [review]
LGTM

**Review by elfredawong — COMMENTED** [review]
1. Could we change this line as well? 
In addition to the configuration, the Securosys’ Primus X JCA provider (version **1.8.2 or newer**) needs to be placed in the node’s drivers folder.

2. Can we review and update the supported version on other page as well? Below are the ones came up if I search "Securosys" but please double check if there is any other pages:

https://docs.r3.com/en/platform/corda/4.9/enterprise/operations/deployment/hsm-deployment.html#securosys-primus-x
https://docs.r3.com/en/platform/corda/4.9/enterprise/operations/deployment/hsm-deployment-confidential.html#additional-notes
https://docs.r3.com/en/platform/corda/4.9/enterprise/node/operating/confidential-identities-hsm.html#additional-notes

File: content/en/platform/corda/4.10/enterprise/platform-support-matrix.md [hunk 1]
```diff
@@ -126,22 +126,22 @@ Install the **Java 8 JDK**. Corda does not currently support Java 9 or higher.
...
-| Gemalto Luna                  | Firmware version 7.0.3   | Firmware version 7.0.3   | Firmware version 7.0.3  | Firmware version 7.0.3    |
-|                               | Driver version 7.3       | Driver version 7.3       | Driver version 7.3      | Driver version 7.3        |
-| FutureX Vectera Plus          | Firmware version 6.1.5.8 | Firmware version 6.1.5.8 | Firmware version 6.1.5.8 | Firmware version 6.1.5.8  |
-|                               | PKCS#11 version 3.1      | PKCS#11 version 3.1      | PKCS#11 version 3.1      | PKCS#11 version 3.1       |
-|                               | FXJCA version 1.17       | FXJCA version 1.17       | FXJCA version 1.17       | FXJCA version 1.17        |
-| Azure Key Vault               | Driver version           | Driver version           | Driver version (SOFTWARE mode only)| Driver version  |
-|                               | azure-identity 1.2.0     | azure-identity 1.2.0     | azure-identity 1.2.0     | azure-identity 1.2.0      |
+|Device                         | Legal identity & CA keys  | TLS keys                | Confidential identity keys | Notary service keys         |
```
Comment (AnnaK-R3) [inline content/en/platform/corda/4.10/enterprise/platform-support-matrix.md:129]:
```suggestion
|Device                         | Legal identity and CA keys  | TLS keys                | Confidential identity keys | Notary service keys         |
```

**Review by colin-hillis — DISMISSED** [review]
LGTM

File: content/en/platform/corda/4.10/enterprise/platform-support-matrix.md (hunk 1 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.10/enterprise/platform-support-matrix.md:129]: Done (in three places)

**Review by AnnaK-R3 — APPROVED** [review]

**Pull request merged**